client/e2e/pages/authenticated/dashboard/profile/page.spec.ts

```diff
@@ -0,0 +1,81 @@
+// 🧪 Suite to test the onboarding\Home page `http://localhost:3000/home`
+
+import { test } from "@playwright/test";
+// 🪄 Page Object Models
+import { DashboardPOM } from "@/e2e/poms/dashboard";
+import { ProfilePOM } from "@/e2e/poms/profile";
+// ☰ Enums
+import { UserRole } from "@/e2e/utils/enums";
+// 🥞 Connection pool to postgres DB
+import { pool } from "@/e2e/utils/pool";
+import { deleteNewUser, upsertIOUser } from "@/e2e/utils/queries";
+// ℹ️ Environment variables
+import * as dotenv from "dotenv";
+dotenv.config({ path: "./e2e/.env.local" });
+
+// 🛠️ Function: deletes the new user record from the database
+const deleteNewUserRecord = async () => {
```

Comment:
Why do we need to delete the new user? If this test doesn't need it, couldn't we just not add it in the first place?

Reply:
@BCerki  The profile update flow differs for a user in the user table and a user not in the user table; so,  ` query = upsertUserIO;` upserts user record for ` bc-cas-dev-secondary`, and `query = deleteUserNew` deletes  user record for ` bc-cas-dev-three` allowing tests for an ID with role `industry_user` and and ID with no role.

Reply:
Still a bit lost--we use `upsertUserIO` to add `bc-cas-dev-secondary`. But how did `bc-cas-dev-three` get into the db?

Reply:
`bc-cas-dev-three` gets into the db via the profile test's update and submit actions: `Test Workflow new user › Test Update Profile`